### ISSUE TITLE
Can GLES 2.0 be Supported?

### ISSUE THREAD

**Issue by darcyg** [eXeC64/imv#204]
Dear Harry
It needs to be used on raspberry pie. But now IMV is based on standard GL. Many macros are not supported. Hope to provide OpenGLES 2.0 compatible method.

thanks

**Comment by eXeC64** [comment]
Yes, that seems reasonable. It might be a while before I work on this though, so volunteers welcome.
